packages/server/lib/errors.ts

```diff
@@ -1,1246 +1,10 @@
+import errors from '@packages/errors'
```

Comment:
did you mean to delete this file?

Reply:
No, it's meant to keep some of the existing tests & exist code in place without needing to refactor